src/app/PriceChart.tsx

```diff
@@ -0,0 +1,265 @@
+import { createChart, CandlestickData, UTCTimestamp } from "lightweight-charts";
+import React, { useEffect, useRef, useContext, useState } from "react";
+import { AdexStateContext } from "./page";
+import * as adex from "alphadex-sdk-js";
+
+// from https://www.npmjs.com/package/alphadex-sdk-js#candleperiods-array
+const CANDLE_PERIODS = ["5m", "15m", "1h", "4h", "6h", "12h", "1D", "1W", "1M"];
```

Comment:
@fliebenberg are the periods available via AlphaDEX SDK somewhere? Maybe an array like this, or an enum?

Reply:
Yes, they are available on the SDK. You can get then like this: 
const CANDLE_PERIODS = adex.CandlePeriods; 
Or you can of course just use adex.CandlePeriods straight in your code.